development.txt

```diff
@@ -0,0 +1,99 @@
+Instructions for creating a local development environment to build/test features.
```

Comment:
Thanks for putting this together! I'm eager to try this on my own so I'm not always testing things on the production app 😆